File: solutions/beecrowd/1007/1007.cpp
#include <cstdint>
#include <cstdio>

int main() {
    std::int16_t a, b, c, d;

    scanf("%d", &a);
    scanf("%d", &b);
    scanf("%d", &c);
    scanf("%d", &d);

    printf("DIFERENCA = %d\n", a * b - c * d);

    return 0;
}

File: solutions/beecrowd/1007/1007.go
package main

import "fmt"

func main() {
	var a, b, c, d int16

	fmt.Scanf("%d", &a)
	fmt.Scanf("%d", &b)
	fmt.Scanf("%d", &c)
	fmt.Scanf("%d", &d)

	fmt.Printf("DIFERENCA = %d\n", a*b-c*d)
}
